src/rwa/components/table/assets/assets-table.tsx

```diff
@@ -219,12 +214,10 @@ export function AssetsTable(props: AssetsTableProps) {
                         </RWATableCell>
                     )}
                     {column.key !== 'name' &&
-                        column.key !== 'notional' &&
                         column.key !== 'purchaseProceeds' &&
                         column.key !== 'salesProceeds' &&
                         column.key !== 'totalDiscount' &&
-                        column.key !== 'realizedSurplus' &&
-                        column.key !== 'currentValue' && (
```

Comment:
Was the currentValue deleted on purpose?

Reply:
yes, I think it wasn't removed previously. This was causing a missing cell.

Reply:
But the currentValue is still shown with this change?

Reply:
yes, this is for the totals row. Previously we were rendering null for `currentValue` total cell, causing an offset (the rest of total values where displayed in the incorrect column). now `currentValue` is handled by this last change and renders an empty cell (just like all the other cells that does not require a total).